invenio_users_resources/services/schemas.py

```diff
@@ -52,6 +52,9 @@ class UserSchema(BaseRecordSchema, FieldPermissionsMixin):
         "active": "read_details",
         "confirmed": "read_details",
         "preferences": "read_details",
+        "blocket_at": "read_details",
```

Comment:
This means that the user itself can search for these. 
We might want to have something more exclusive to make it only visible to administrators